Makefile

```diff
@@ -524,8 +526,20 @@ out/linters/golangci-lint-$(GOLINT_VERSION):
 .PHONY: lint
 ifeq ($(MINIKUBE_BUILD_IN_DOCKER),y)
 lint:
-	docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:$(GOLINT_VERSION) \
-	golangci-lint run ${GOLINT_OPTIONS} --skip-dirs "cmd/drivers/kvm|cmd/drivers/hyperkit|pkg/drivers/kvm|pkg/drivers/hyperkit" ./...
+	mkdir -p ./out/linters
+	@if [ ! -f ./out/linters/yq ] ; then \
+		wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY_NAME} -O ./out/linters/yq && chmod +x ./out/linters/yq; \
```

Comment:
this doesnt seem right to install things using wget, and we we need to install this ?